battle.net/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,26 @@
+$ErrorActionPreference = 'Stop';
+$toolsDir     = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$ahkExe       = 'AutoHotKey'
+$ahkFile      = "$toolsDir\battle.net_install.ahk"
+$url          = 'https://us.battle.net/download/getInstaller?os=win&installer=Battle.net-Setup.exe&id=undefined'
+$fileLocation = Join-Path $toolsDir 'Battle.net-Setup.exe'
+$checksumType = 'sha256'
+
+Invoke-WebRequest $url -OutFile $fileLocation -UseBasicParsing

```

Comment:
There is not really any point in downloading the file beforehand to get the checksum for `Install-ChocolateyPackage`, at least security wise.